source/features/html-preview-link.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import features from '../libs/features';
+import {groupSiblings} from '../libs/group-buttons';
+
+function init(): void {
+	const rawButton = select('#raw-url') as HTMLAnchorElement;
```

Comment:
You can use `select<HTMLAnchorElement>(...)!;

Reply:
😅 `rawButton.pathname` is used below where `pathname` is from `HTMLAnchorElement`